trestle/core/commands/author/profile.py

```diff
@@ -286,30 +286,23 @@ def _replace_modify_set_params(
         return changed
 
     @staticmethod
-    def _add_aggregated_parameter(
-        param: Any, param_dict: Dict[str, Any], control_id: str, controls: Any, param_map: Dict[str, str]
-    ) -> None:
+    def _add_aggregated_parameter(param: Any, param_dict: Dict[str, Any]) -> None:
```

Comment:
Was aggregated parameter already handled before? if yes was was the code doing for those params.